src/core/ColorPicker/ColorInputPanel.tsx

```diff
@@ -86,6 +103,7 @@ export const ColorInputPanel = React.forwardRef(
 
     const handleColorInputChange = () => {
       let color;
+      let alpha = 1;
 
       if (currentFormat === 'hex') {
```

Comment:
Might need some special handling to support HEXA strings

Reply:
updated HEXA input